source/documentation/deploying_apps/deploying_django.md

```diff
@@ -2,7 +2,7 @@
 
 This section explains how to deploy an app using the Django framework. You may also need to refer to the [Cloud Foundry documentation about the Python buildpack](https://docs.cloudfoundry.org/buildpacks/python/index.html) [external link].
 
-Note that the only database service currently supported by PaaS is PostgreSQL. If your Django app requires a database, it must be able to work with PostgreSQL.
+> If your app requires a [backing service](https://docs.cloud.service.gov.uk/#deploy-a-backing-or-routing-service), it must be able to work with one of the services supported by PaaS. Instructions for deploying both backing service and non-backing service apps are given in this section.
```

Comment:
Could you change all of the links in this PR to be root-relative rather than absolute? For example `/#deploy-a-backing-or-routing-service` instead of `https://docs.cloud.service.gov.uk/#deploy-a-backing-or-routing-service`. This will ensure that the links work when the site is deployed somewhere else, such as previewing locally.